dev-tools/mage/dockerbuilder.go

```diff
@@ -178,6 +178,10 @@ func (b *dockerBuilder) expandDockerfile(templatesDir string, data map[string]in
 
 func (b *dockerBuilder) dockerBuild() (string, error) {
 	tag := fmt.Sprintf("%s:%s", b.imageName, b.Version)
+	// For Independent Agent releases, replace the "+" with a "." since the "+" character
+	// currently isn't allowed in a tag in Docker
+	// E.g., 8.13.0+build202402191057 -> 8.13.0.build202402191057
+	tag = strings.Replace(tag, "+", ".", 1)
```

Comment:
I have a question, if `b.Snapshot` is defined then the resulting tag would be 
`8.13.0.build202402191057-SNAPSHOT` is this the expected behavior for a snapshot build, or is this scenario never going to happen?

Reply:
We only plan on having a Staging build for the Independent Agent Release.  No Snapshot builds.